cmd/server/handler.go

```diff
@@ -154,6 +154,29 @@ func RunServeAll(version, build, date string) func(cmd *cobra.Command, args []st
 	}
 }
 
+func setTracingLogger(d driver.Driver, logger *negronilogrus.Middleware) {
+	headers := make(map[string]string)
+
+	if d.Configuration().LogHeaderTraceID() != "" {
+		headers["trace_id"] = d.Configuration().LogHeaderTraceID()
+	}
+	if d.Configuration().LogHeaderSpanID() != "" {
+		headers["span_id"] = d.Configuration().LogHeaderSpanID()
+	}
+
+	if len(headers) == 0 {
+		return
+	}
+
+	logger.Before = func(entry *logrus.Entry, r *http.Request, _ string) *logrus.Entry {
+		fields := make(map[string]interface{})
+		for key, headerKey := range headers {
+			fields[key] = r.Header.Get(headerKey)
```

Comment:
This does not be to be compliant with the open tracing specification: https://github.com/opentracing/specification/blob/584c52365fe0700ba25effa79dce5effb845ced8/rfc/trace_identifiers.md

I would expect that this parses the HTTP Header `traceparent`. I'm pretty sure that opentracing has a library to do that, so maybe we can just use that here?

Reply:
You're right. And I found library to extract tracing information from header!
https://github.com/open-telemetry/opentelemetry-go